Dockerfile

```diff
@@ -78,14 +78,14 @@ RUN sed -i '/^av==/d' /tmp/utils/dataset_manifest/requirements.txt
 
 ARG CVAT_CONFIGURATION="production"
 
-RUN --mount=type=cache,target=/root/.cache/pip/http \
+RUN --mount=type=cache,target=/root/.cache/pip/http-v2 \
     DATUMARO_HEADLESS=1 python3 -m pip wheel --no-deps \
     -r /tmp/cvat/requirements/${CVAT_CONFIGURATION}.txt \
     -w /tmp/wheelhouse
 
-FROM golang:1.20.5 AS build-smokescreen
+FROM golang:1.22.3 AS build-smokescreen
 
-RUN git clone --depth=1 -b v0.0.4 https://github.com/stripe/smokescreen.git
+RUN git clone --depth=1 https://github.com/stripe/smokescreen.git
```

Comment:
You have to check out a specific commit; otherwise, the builds will not be reproducible.

Reply:
Done